src/main/webapp/app/view/project/admin/project-admin.module.ts

```diff
@@ -26,6 +26,7 @@ import { ProjectResponsibilityUpdateComponent } from 'app/view/project/admin/res
 import { ProjectResponsibilityDeleteDialogComponent } from 'app/view/project/admin/responsibilities/project-responsibility-delete-dialog.component';
 
 import { PROJECT_ADMIN_ROUTES } from './project-admin.routes';
+import { ProjectUserInviteDeleteDialogComponent } from 'app/view/project/admin/users/project-user-invite-delete-dialog.component';
```

Comment:
import after `ProjectUserInviteComponent`.

Reply:
du bist wahrscheinlich die einzige person, die auf die reihenfolge von imports schaut :D

Reply:
it’s the little things that matter the most.